### ISSUE TITLE
Weapons handling, manual reload, laser dot

### ISSUE THREAD

**Issue by benjamincheng10** [fholger/farcry_vrmod#17]
Thanks for including the option to adjust weapon tilt, that helps a lot! I noticed that the laser dot doesn't match the weapon sights when they are adjusted, could the laser be made to line up with the weapon model?
 
maybe an option to turn off the laser dot?

any plans to add manual reloading for weapons?

since you can only carry 4 weapons, toggling with a click is not so bad, a selection menu like in HL2VR would be better IMO

really clever the way the scoped weapons are handled, only annoyance is that the scope UI is horizontally squished. not big deal tho.

**Comment by gawaines71** [comment]
Awesome job, big thanks.
As it´s been said, the laser dot should be optional, most of the times it doesn´t even match the sights and, being in vr,  when it does it feels a little bit like cheating.

Thanks again.

**Comment by fholger** [comment]
That's the point, though - the sights are *not* accurate on most of the weapon models. So you can't aim them properly without the dot unless the models are properly overhauled. Until that happens (which I can't do, I'm not a modeller), I'm afraid aiming by sight is simply not possible in this mod.

**Comment by VinceCrusty** [comment]
Of course, it would be nice if you could replace the dot with a normal crosshair in red or a red dot. The blue dot looks very ugly and is much too big. Unfortunately, I have not been able to discover the dot in the assets so far, or it does not consist of a texture.

Sonst ein großartiger Mod, gratulation 😋👍

**Comment by gawaines71** [comment]
I see. Maybe as a lesser evil it could be turned into a minute red dot.
Thanks for the mod.

**Comment by fholger** [comment]
Not as easily as you might think. I'm using what is effectively a debug draw utility in the engine as a stop gap for the crosshair, and it gives exactly what we have - a slightly blue ball at the position I place it at. I have no control over the color; I could make it smaller, but that will also make it a lot harder to see in some situations.

Obviously, it's not meant to be the final crosshair representation. But a better implementation will require some extra work on my part, and it's just not a high enough priority right now compared to other things I want to address.

**Comment by VinceCrusty** [comment]
Good to know, thanks for the explanation. Thanks anyway!

**Comment by benjamincheng10** [comment]
Thanks for included some laser options in the latest version! the OICW is usable now! I noticed that the scope image is much closer than it used to be for all the weapons, making it hard to focus on. can we have an option for it to be further out? Also the Binoc controls on the offhand feels really awkward for me, i understand it might be good for motion sickness? I liked the original headtracked Binoc, can we have that option also?

**Comment by fholger** [comment]
I'm afraid not. The original solution was a stop gap measure, this is what I actually wanted it to be like. Having options for every single thing in the game is a hassle to implement and maintain, so I do not currently have plans to support both.

**Comment by benjamincheng10** [comment]
The weapon scopes still feels a bit too close but if I extend my off hand it feels fine. on the OICW the scope doesn't activate until the gun clips into the camera tho, feels bit uncomfortable. getting used to the way the Binoc works now, although the angle of it is a bit uncomfortable. can we get the binoc view plane tilted up 30-45 degrees so its at a more comfortable angle to hold?
